### ISSUE TITLE
Sudo shutdown 

### ISSUE THREAD

**Issue by ashwinvasudevan** [mozilla/TBTWBuildChallenge#14]
# Build Technology Idea Submission

_Please answer all of the questions below._
- Team Name: Easwari Engineering Collge, Firefox Students group
- Challenge Contact Person Name: Ashwin Vasudevan
- Campus Name: Easwari Engineering College
### Idea Description

_Please answer all of the questions below._
- Idea name: sudo shutdown
- What is the problem you are solving for:  Peter Sunde, founder of Pirate Bay, was arrested by Police in May 2014. Before he was arrested, he was able to open up an app on his phone and shutdown down all his computers. Since the computer was encrypted, the police were unable to obtain any information from his computers. **Sudo Shutdown** is an app which will be developed to send remote shutdown commands to all computers. Provided that the partition is encrypted, data cannot be obtained. 
- Who encounters that problem (who is your audience - describe them): Media persons, open source activists, Journalists, Whistle Blowers, people with highly confidential data.
- How will your tool solve this problem for these people:  Our app, on touch of a button will issue shutdown command to all the computers involved. Our app will also provide instructions on how to properly secure your computer using encryption. 
- What are you going to create (app, add-on, extension, etc.):  App
- What are the next steps for building this tool: Development of the app.
- Links to additional information: 

https://www.theguardian.com/technology/2014/nov/05/sp-pirate-bay-cofounder-peter-sunde-in-prison

**Comment by LucyeoH-zz** [comment]
Reviewers: @acabunoc @ScottDowne

**Comment by abbycabs** [comment]
Hi @ashwinvasudevan! Interesting idea. What exactly will the shut down command do? Turn off all your devices remotely? Are you expecting your users to already know how to encrypt their machines?

Thanks for the idea proposal!

**Comment by ashwinvasudevan** [comment]
@acabunoc The app will shutdown all the devices you have linked remotely. 
We will be providing users instructions on how to encrypt their devices if they have not already done so.

**Comment by abbycabs** [comment]
This is not very useful if users don't know about encryption. I you're already planning on writing instructions, I think it might be a good idea to focus on developing the encryption lessons.

**Comment by ashwinvasudevan** [comment]
@acabunoc Target audience are media persons, journalists, Whistle Blowers, people with highly confidential data. Instructions are just for recommending best practises and such.

**Comment by charissalaisy** [comment]
@ashwinvasudevan, thanks so much for this submission! We’re excited to hear about your idea and hope you’ll take the feedback you got to go on to the next phase by turning this into a prototype.

A prototype can come in many different forms, some of which you can read about [here](https://hackdesign.org/lessons/10).

Over the next few weeks we will share a number of resources to help you build your prototype.

Once your prototype is ready please link your picture, slideshow, code right here in your issue by June 10th.

We can’t wait to see what you create!

**Comment by ashwinvasudevan** [comment]
https://github.com/ashwinvasudevan/TBTWC-Mozilla @charissalaisy
